nix/vast/default.nix

```diff
@@ -12,6 +12,7 @@
 , libpcap
 , arrow-cpp
 , flatbuffers
+, spdlog
```

Comment:
You also need to add `spdlog` to the `buildInputs` list on line 71.

Reply:
Thanks!